sydent/http/httpclient.py

```diff
@@ -59,7 +43,12 @@ def get_json(self, uri):
             uri.encode("ascii"),
         )
         body = yield readBody(response)
-        defer.returnValue(json.loads(body))
+        try:
+            json_body = json.loads(body)
+        except Exception as e:
+            logger.exception("Error parsing JSON from %s", uri)
+            json_body = None
```

Comment:
I'm not sure that returning None in this case is correct. we should raise an exception of some sort, possibly the original exception...